docs/api/scripts_kg_modules.md

```
@@ -0,0 +1,258 @@
+# `scripts/kg/modules`
+
+
+## `scripts\kg\modules\__init__`
+
+**🧠 Docstring Summary**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)